ee/libcglue/src/fdman.h

```diff
@@ -14,7 +14,7 @@
 
 #define _FDMAN_H_
 
-#define __FILENO_MAX 1024
+#define __FILENO_MAX 64
```

Comment:
This is crucial to save RAM, what do you think should be a good number here?
I would say that 64 is fair enough....